crt_portal/static/js/dashboards-index-page.js

```diff
@@ -0,0 +1,182 @@
+    );
+    sectionDashboardsCardsContainerElement.innerHTML = '';
+    dashboardsList.slice(startIndex, endIndex).forEach(dashboardData => {
+      sectionDashboardsCardsContainerElement.appendChild(buildDashboardCardHTML(dashboardData));
+    });
+
+    const sectionPaginationContainer = document.getElementById(
+      section + '-dashboards-pagination-container'
+    );
+    const pageNumberElement = document.getElementById(section + '-dashboards-current-page');
+    sectionPaginationContainer.style.display = 'none';
+    pageNumberElement.style.display = 'none';
+    if (numberOfPages != 1) {
+      sectionPaginationContainer.style.display = 'flex';
+      const lastPageElement = document.getElementById(section + '-dashboards-last-page');
+      lastPageElement.textContent = numberOfPages.toString();
+
+      const displayedPageNumber = pageNumber + 1;
+      pageNumberElement.textContent = displayedPageNumber.toLocaleString();
+      pageNumberElement.style.display = 'flex';
+    }
+
+    const paginationPrevious = document.getElementById(section + '-dashboards-previous');
+    paginationPrevious.style.display = 'none';
+    if (pageNumber > 0) {
+      paginationPrevious.style.display = 'flex';
+    }
+
+    const paginationNext = document.getElementById(section + '-dashboards-next');
+    paginationNext.style.display = 'none';
+    if (pageNumber < numberOfPages - 1) {
+      paginationNext.style.display = 'flex';
+    }
+    sectionDashboardsCardsContainerElement.dataset.currentPage = pageNumber;
+    sectionDashboardsCardsContainerElement.dataset.lastPage = numberOfPages - 1;
+  }
+
+  function handleFilterBySection(section) {
+    const containers = document.getElementsByName('section-dashboards-container');
+    for (const containerElement of containers) {
+      if (!section) {
+        containerElement.style.display = 'block';
+        continue;
+      }
+
+      if (containerElement === document.getElementById(section + '-dashboards-container')) {
+        containerElement.style.display = 'block';
+      } else {
+        containerElement.style.display = 'none';
+      }
+    }
+  }
+
+  function handleDashboardsPaginationNextPrevious(section, forward = true) {
+    const dashboardDataBySection = JSON.parse(
+      document.getElementById('dashboards-by-section-data').textContent
+    );
+    const dashboardsList = dashboardDataBySection[section];
+
+    const sectionDashboardsCardsContainerElement = document.getElementById(
+      section + '-dashboards-cards-container'
+    );
+    const currentPageNumber = parseInt(sectionDashboardsCardsContainerElement.dataset.currentPage);
+    const pageSize = parseInt(sectionDashboardsCardsContainerElement.dataset.pageSize);
+
+    if (forward) {
+      paginateSectionDashboards(section, dashboardsList, pageSize, currentPageNumber + 1);
+    } else {
+      paginateSectionDashboards(section, dashboardsList, pageSize, currentPageNumber - 1);
+    }
```

Comment:
nit: to avoid the duplicate function call:

```suggestion
    const direction = forward ? 1 : -1;
    paginateSectionDashboards(section, dashboardsList, pageSize, currentPageNumber + direction);
```